doc/sphinx/source/data/new-commondata.rst

```diff
@@ -0,0 +1,155 @@
+Organization following the naming convention
+--------------------------------------------
+
+All dataset in the new data format follow the exact same naming convention::
+
+    <experiment>_<process>_<energy>{_<extras>}_<observable>
+
+The data is contained in folders, each containing one single hepdata publication. 
+In all cases one can reconstruct the name of the folder by separating the observable name on the last ``_``, i.e., the folder will always be named::
+
+    <experiment>_<process>_<energy>{_<extras>}
+
+
+Where all observables contained in one hepdata entry are separated by their observable name.
+
+Metadata Format
```

Comment:
At some point before here we should discuss what is required (metadata, kinematics) and what is optional (everything else).

Reply:
I don't agree with the everything else. I think hepdata is required and then if it is empty one can add "NA" or whatever. Same for version. Or maybe you are referring to that with `metadata`.

The reason is that I would like to avoid a situation in which a dataset is implemented (or amended) with that information missing even if the cost is a slightly more cumbersome implementation for say a positivity dataset.